setup.py

```diff
@@ -7,7 +7,7 @@
 
 dask_deps = ["dask>=2.22.0", "distributed>=2.22.0"]
 # ray==2.5.0 broken: https://github.com/conda-forge/ray-packages-feedstock/issues/100
-ray_deps = ["ray[default]>=2.1.0,!=2.5.0", "pyarrow>=7.0.0"]
+ray_deps = ["ray>=2.1.0,!=2.5.0", "pyarrow>=7.0.0", "grpcio"]
```

Comment:
```suggestion
ray_deps = ["ray-core>=2.1.0,!=2.5.0", "pyarrow>=7.0.0", "grpcio"]
```

Reply:
Analogue of conda package `ray-core` for pip - just `ray`.

Reply:
ping

Reply:
I don't think so, this knowledge is easily obtained from the ray installation page (if needed).